gui/src/serveritemwidget.cpp

```diff
@@ -40,6 +40,12 @@ ServerItemWidget::ServerItemWidget(QWidget *parent) : QFrame(parent)
 	addAction(wake_action);
 	connect(wake_action, &QAction::triggered, this, [this]{ emit WakeTriggered(); });
 
+	#if defined(__linux) || (defined(__APPLE__) && defined(CHIAKI_ENABLE_CLI))
+		create_shortcut_action = new QAction(tr("Add to Steam"), this);
+		addAction(create_shortcut_action);
+		connect(create_shortcut_action, &QAction::triggered, this, [this]{ emit CreateShortcutTriggered(); });
```

Comment:
`connect(create_shortcut_action, &QAction::triggered, this, &ServerItemWidget::CreateShortcutTriggered);`